tests/containers/podman_integration.pm

```diff
@@ -19,6 +19,28 @@ use containers::common;
 
 my $test_dir = "/var/tmp";
 my $podman_version = "";
+my $quadlet = "";
```

Comment:
I would not use a global variable. Either pass it as subroutine parameter or evaluate it every time (which is ok here).

Reply:
Fixed.